rootski_api/README.rst

```diff
@@ -1 +1,61 @@
-FastAPI Backend for Rootski
+=====================================================
```

Comment:
Interesting, I don't think this file was supposed to be committed. Can you delete the contents of this file and commit that deletion? I think it's fine if the file itself exists, but is empty.